python/dolfinx/wrappers/fem.cpp

```diff
@@ -1007,6 +1007,15 @@ void fem(py::module& m)
   m.def("transpose_dofmap", &dolfinx::fem::transpose_dofmap,
         "Build the index to (cell, local index) map from a "
         "dofmap ((cell, local index ) -> index).");
+  // FIXME Where is the best place to put this?
+  m.def(
+      "compute_integration_domains",
```

Comment:
This will need a more elaborate wrapper to return `py::array_t`s rather than `std::vector`